packages/manager/apps/hycu/public/translations/hycu/dashboard/Messages_fr_FR.json

```diff
@@ -24,8 +24,9 @@
   "hycu_dashboard_wait_for_activation": "En attente d'activation",
   "hycu_dashboard_license_activate_heading": "Activer la licence",
   "hycu_dashboard_license_activate_description": "Veuillez transférer votre fichier de demande de licence. Ce fichier est à télécharger depuis votre controller HYCU une fois votre installation réalisée",
-  "hycu_dashboard_license_more_information_link": "(plus de détails ici)",
+  "hycu_dashboard_license_more_information_link": "(plus de détails ici).",
   "hycu_dashboard_activation_license_success_message": "La demande d'activation de votre licence a bien été prise en compte. Votre fichier de licence sera bientôt disponible sur cette interface et vous sera également envoyé par e-mail.",
+  "hycu_dashboard_regenerate_success_message": "La demande d'activation de votre licence a bien été prise en compte. Votre fichier de licence sera bientôt disponible sur cette interface et vous sera également envoyé par e-mail.",
```

Comment:
Duplicate translation

Reply:
@tibs245 Can you use the same key for both use cases